citrix_hypervisor/datadog_checks/citrix_hypervisor/check.py

```diff
@@ -47,12 +45,11 @@ def _get_updated_metrics(self):
             'host': 'true',
             'json': 'true',
         }
+        # by default, returns json response
         r = self.http.get(self._base_url + '/rrd_updates', params=params)
         r.raise_for_status()
-        # Response is not formatted for simplejson, it's missing double quotes " around the field names
-        # Explicitly use the python safe loader, the C binding is failing
-        # See https://github.com/yaml/pyyaml/issues/443
-        data = yaml.load(r.content, Loader=yaml.SafeLoader)
+
+        data = r.json()
```

Comment:
I am wondering, why is it that we were not doing this before and now we can? Just making sure this is not going to fix tests but fail when running the integration with the real system. If we are covered by any integration/e2e I am ok with it.